src/nvim/fold.c

```diff
@@ -282,10 +276,11 @@ int foldLevel(linenr_T lnum)
   return foldLevelWin(curwin, lnum);
 }
 
-// lineFolded() {{{2
-// Low level function to check if a line is folded.  Doesn't use any caching.
-// Return true if line is folded.
-// Return false if line is not folded.
+/// lineFolded() {{{2
```

Comment:
Comments like this do not seem like documentation comments.

Reply:
You're right. Thanks.